packages/blade/src/components/FileUpload/FileUpload.web.tsx

```diff
@@ -354,6 +354,119 @@ const _FileUpload: React.ForwardRefRenderFunction<BladeElementRef, FileUploadPro
   );
 };
 
+/**
+ * ### FileUpload Component
+ * 
```

Comment:
This is a little too large for jsdoc 🙈 Its hard to parse through this much code in small dialog box of jsdoc. Just keep `<FileUpload />` component with its props maybe? ignore the state management and surrounding box buttons etc. Doesn't have to be a working example

Reply:
Updated but in https://github.com/razorpay/blade/pull/2040